compute/cloud-client/src/main/java/compute/disks/CreateDiskSecondaryRegional.java

```diff
@@ -0,0 +1,106 @@
+    // https://cloud.google.com/compute/docs/disks/async-pd/about#supported_region_pairs
+    String primaryDiskRegion = "us-central1";
+    // Name of the region in which you want to create the secondary disk.
+    String secondaryDiskRegion = "us-east1";
+    // Size of the new disk in gigabytes.
+    // Learn more about disk requirements:
+    // https://cloud.google.com/compute/docs/disks/async-pd/configure?authuser=0#disk_requirements
+    long diskSizeGb = 30L;
+    // The type of the disk you want to create. This value uses the following format:
+    // "projects/{projectId}/zones/{zone}/diskTypes/
+    // (pd-standard|pd-ssd|pd-balanced|pd-extreme)".
+    String diskType = String.format(
+         "projects/%s/regions/%s/diskTypes/pd-balanced", secondaryProjectId, secondaryDiskRegion);
+
+    createDiskSecondaryRegional(primaryProjectId, secondaryProjectId, primaryDiskName,
+            secondaryDiskName, primaryDiskRegion, secondaryDiskRegion, diskSizeGb, diskType);
+  }
+
+  // Creates a secondary disk in a specified region.
+  public static Disk createDiskSecondaryRegional(String projectId, String secondaryProjectId,
+       String primaryDiskName, String secondaryDiskName, String primaryDiskRegion,
+       String secondaryDiskRegion, long diskSizeGb,  String diskType)
+        throws IOException, ExecutionException, InterruptedException, TimeoutException {
+    // An iterable collection of zone names in which you want to keep
+    // the new disks' replicas. One of the replica zones of the clone must match
+    // the zone of the source disk.
+    List<String> replicaZones = Arrays.asList(
+            String.format("projects/%s/zones/%s-c", secondaryProjectId, secondaryDiskRegion),
+            String.format("projects/%s/zones/%s-b", secondaryProjectId, secondaryDiskRegion));
+
+    String primaryDiskSource = String.format("projects/%s/regions/%s/disks/%s",
+            projectId, primaryDiskRegion, primaryDiskName);
+
+    DiskAsyncReplication asyncReplication = DiskAsyncReplication.newBuilder()
+            .setDisk(primaryDiskSource)
+            .build();
+
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests.
+    try (RegionDisksClient disksClient = RegionDisksClient.create()) {
+
+      Disk disk = Disk.newBuilder()
+          .addAllReplicaZones(replicaZones)
+          .setName(secondaryDiskName)
+          .setSizeGb(diskSizeGb)
+          .setType(diskType)
+          .setRegion(secondaryDiskRegion)
+          .setAsyncPrimaryDisk(asyncReplication)
+          .build();
+
+      // Wait for the create disk operation to complete.
+      Operation response = disksClient.insertAsync(secondaryProjectId, secondaryDiskRegion, disk)
+          .get(3, TimeUnit.MINUTES);
+
+      if (response.hasError()) {
+        return null;
+      }
+      return disksClient.get(secondaryProjectId, secondaryDiskRegion, secondaryDiskName);
```

Comment:
insertAsync returns OperationFuture<Operation, Operation>. In other samples with create operation Sita asked me to return resource that was created and then test it, instead of printing a message.